indexing.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `r_tb_assay` ADD INDEX( `id`,`short_name`);
+ALTER TABLE `reference_result_tb` ADD INDEX( `shipment_id`,`sample_id`,`is_excluded`);
+ALTER TABLE `response_result_tb` ADD INDEX( `shipment_map_id`,`sample_id`);
+ALTER TABLE `shipment` ADD INDEX( `shipment_id`);
+ALTER TABLE `shipment_participant_map` ADD INDEX( `attributes`,`map_id`,`is_excluded`);

```

Comment:
Indexing `attributes` is not going to help. If anything, it will just add extra overhead to write operations and slow the system down a bit.

I was hoping that you would add a full create database script that includes indexes, that would be useful for anybody who needs to create the database in their local environment or to perform DB structure comparisons between production and staging environments.

From now on, please can we try to separate tickets into their own PRs? Smaller changes are much easier to review.

Reply:
I have removed the indexing file from this PR/Branch and will create new branch for database indexing issue.